player/playloop.c

```diff
@@ -665,6 +666,7 @@ static void handle_osd_redraw(struct MPContext *mpctx)
     if (!want_redraw)
         return;
     vo_redraw(mpctx->video_out);
+    refresh_subs(mpctx);
```

Comment:
Don't you want to refresh subs, before redraw starts?

Reply:
So this is kind of a bad name. Even though this is indeed a redraw, we actually want to redraw again but in the loop in player/sub. Strictly speaking, the order here doesn't matter. It can be before/after. All this does is set a flag that gets used later.

Reply:
Renamed it to `redraw_subs` and added a comment to be clearer.